AccelGyro accelGyro
float64[9] magnetic_field_covariance
